internal/cmd/plugin/install_utils.go

```diff
@@ -0,0 +1,134 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/exec"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+var supportedLanguages = []string{"python", "go"}
+
+const (
+	programNotFoundMsg      = "[WARN] Unable to find %s. Check that it is installed in a directory in your $PATH.\n"
+	unableToParseVersionMsg = "[WARN] Unable to parse %s version.\n"
+	insufficientVersionMsg  = "[WARN] Installed %s version %s is less than the required version %s.\n"
+)
+
+func getLanguage(manifest *Manifest) (string, *version.Version, error) {
+	if manifest == nil {
+		return "", nil, nil
+	}
+
+	if manifest.Dependencies == "" {
+		return "", nil, nil
+	}
+
+	dependencySlice := strings.Split(strings.ToLower(manifest.Dependencies), " ")
+	if len(dependencySlice) == 1 {
+		return dependencySlice[0], nil, nil
```

Comment:
What is `dependencySlice[0]` and what is `dependencySlice[1]`? Can we put these into variables with better names?

Reply:
The `[0]` entry is the first dependency name, the `[1]` could be either another dependency name or a version number for the first one. Basically, this whole block is me trying to guess what might be in here.

It's why I made that other PR, and I'm going to be rewriting this part anyway to match the new manifest file structure.